src/index.ts

```diff
@@ -402,3 +451,75 @@ extendEnvironment(hre => {
         }
     };
 });
+
+task("starknet-verify", "Verifies the contract in the Starknet network.")
+    .addOptionalParam("starknetNetwork", "The network version to be used (e.g. alpha)")
+    .addParam("path", `The path of the cairo contract (e.g. contracts/conract.cairo)`)
+    .addParam("address", `The address where the contract is deployed`)
+    .setAction(async (args, hre) => {
+        var voyagerUrl = VOYAGER_GOERLI_CONTRACT_API_URL;
+        
+        if(args.starknetNetwork && args.starknetNetwork !== ALPHA){
+            if(args.starknetNetwork === ALPHA_MAINNET)
+                voyagerUrl = VOYAGER_MAINNET_CONTRACT_API_URL;
+            else{
+                const msg = `Unknown starknet-network provided: ${args.starknetNetwork}`;
+                throw new HardhatPluginError(PLUGIN_NAME, msg);
+            }
+        }
+        voyagerUrl+=args.address + "/code";
+        var isVerified = false;
+        await axios.get(voyagerUrl,{
```

Comment:
Why use both await and callbacks?
How about this:
```typescript
const resp = await axios.get(url, args);
const data = resp.data;
```
Or something like this: https://github.com/axios/axios#typescript